about-peer-review/how-peer-review-works.md

```diff
@@ -0,0 +1,47 @@
+# How pyOpenSci's open software peer review works
+
+
+## Who submits packages and who runs the reviews 
+
+pyOpenSci's [suite of packages](https://www.pyopensci.org/python-packages/) are 
+contributed by community members with a great diversity of skills. This diversity 
```

Comment:
```suggestion
contributed by community members with a great diversity of skills and wide range of backgrounds
```
rephrase to avoid repeating "diversity"? I understand the goal here to be to show how we have developers with *both* diversity in skills and backgrounds (which are not the same thing)